reaper/client_test.go

```diff
@@ -101,18 +103,18 @@ func prepareEnvironment(t *testing.T, parent context.Context) {
 	}
 }
 
-func registerClusters(t *testing.T, parent context.Context) {
+func registerClusters(t *testing.T, parent context.Context, client Client) {
 	addClusterGroup, ctx := errgroup.WithContext(parent)
 	t.Log("adding cluster-1 in Reaper...")
 	addClusterGroup.Go(func() error {
-		if err := testenv.AddCluster(ctx, "cluster-1", "cluster-1-node-0"); err != nil {
+		if err := client.AddCluster(ctx, "cluster-1", "cluster-1-node-0"); err != nil {
```

Comment:
It seems that the `AddCluster` function in util.go is now obsolete and could be deleted.